src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -685,6 +685,23 @@ public MeasureSet associateCmsId(
         qdmMeasureId,
         measureSet.getCmsId());
 
+    String associationSuccessMessage =
+        "Measures with IDs %s, %s are Associated with CMS ID %s on %s.";
```

Comment:
can we make it lil bit more clear about QDM and QICore measures like which id is QDM and which one is QICore

Reply:
Sure I can make the change